static/js/containers/UserMenu.js

```diff
@@ -29,7 +29,12 @@ class LoginButton extends React.Component {
         <SplitButton
           title={title}
           bsStyle="danger"
-          id="logout-button">
+          id="user-menu">
+          <MenuItem
+            href={`/users/${profile.username}`}
+            eventKey="users_page">
```

Comment:
can i ask why this approach is preferred? the logout link uses href. the `LinkContainer` adds a good amount of code. what benefit do we get?